godot-macros/src/class/godot_dyn.rs

```diff
@@ -32,6 +32,24 @@ pub fn attribute_godot_dyn(input_decl: venial::Item) -> ParseResult<TokenStream>
         );
     };
 
+    let mut associated_types = vec![];
+    for impl_member in &decl.body_items {
+        let venial::ImplMember::AssocType(associated_type) = impl_member else {
+            continue;
+        };
+        let Some(type_expr) = &associated_type.initializer_ty else {
+            continue;
+        };
+        let type_name = &associated_type.name;
+        associated_types.push(quote! { #type_name = #type_expr })
+    }
+
+    let associated_impl = if associated_types.is_empty() {
+        None
+    } else {
+        Some(quote! { <#(#associated_types, )*>})
+    };
```

Comment:
"Impl" is a bit misleading, as there's no `impl` block involved.

Also, unless we need to distinguish `Some`/`None` in some logic, godot-rust typically uses the empty token stream.

```suggestion
    let assoc_type_constraints = if associated_types.is_empty() {
        TokenStream::new()
    } else {
        quote! { < #(#associated_types),* > }
    };
```

Please apply suggestion verbatim (also spacing + `,` changed).